client/backend.js

```diff
@@ -188,6 +188,10 @@ module.exports = {
     },
 
     updateAccess(accessId, access) {
+        if (access.customFields) {
+            access.customFields = JSON.stringify(access.customFields);
```

Comment:
Right, there's no way to move it down in the data.
However, you can still do this:

`access.customFields = access.customFields ? JSON.stringify(access.customFields) : undefined;`